index.html

```diff
@@ -134,7 +134,9 @@ <h2 class="player-name-text player3-name">Player 3</h2>
     </article>
     <section class="wheel-container">
     <button class="wheel-btn">
-      SPIN
+      <p class="spin-text">PRESS</p>
```

Comment:
Nice! This should take care of the GitHub issue we have around the spin button be more apparent for users.